### PR TITLE
Hs 210 saving user display name

### PR DESCRIPTION
**Pull request by dlvinh**

### CODE REVIEW

File: lib/data/member.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import 'package:hatspace/data/data.dart';
+
+class Member {
+  final rolesKey = 'roles';
```
Comment (DongNguyenQ) [inline lib/data/member.dart:4]: @dlvinh You should specify data type for code convention and consider changing this field to private (_rolesKey), cause i think we don't need to expose it to outside.

File: lib/data/member.dart [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  final displayNameKey = 'displayName';
```
Comment (DongNguyenQ) [inline lib/data/member.dart:5]: @dlvinh Same as rolesKey

File: lib/data/member.dart [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  final Set<Roles> listRoles;
+  final String displayName;
+
+  Member({required this.listRoles, required this.displayName});
+  convertToMap() {
```
Comment (DongNguyenQ) [inline lib/data/member.dart:10]: @dlvinh 
- You should specify data type for code convention, smt like Map<String, dynamic> convertToMap.

File: lib/data/member.dart (hunk 1 above)
Comment (dlvinh) [reply lib/data/member.dart:4]: Thank you @DongNguyenQ, already changed

File: lib/data/member.dart (hunk 2 above)
Comment (dlvinh) [reply lib/data/member.dart:5]: changed as request

**Comment by DongNguyenQ** [comment]
LGTM

**Review by suesitran — APPROVED** [review]

**Pull request merged**